yarn-completion.bash

```diff
@@ -902,7 +908,11 @@ _yarn_workspace() {
 					declare workspace_path
 					declare -a workspaces=()
 					for workspace_path in $(__yarn_get_package_fields -t array workspaces); do
-						workspaces+=("$(basename "$workspace_path")")
+						declare cwd=$PWD
+						until [[ -f "$cwd/package.json" ]]; do
```

Comment:
I had to add this in order to allow users whose `PWD` is `root/packages` to still utilize the `yarn workspace` command with the correct autocomplete suggestions

```
root
- packages
-- package-1
-- package-2
```